README.md

```diff
@@ -134,9 +131,7 @@ config:
 useExistingTunnels:
   enabled: true
   credentialsSecretName: my-tunnel-credentials
-  tunnelIDs:
-    - "f8c06a8a-1880-4e6e-8502-deb8f1d1253b"
-    - "a6392a8a-4dfa-4df6-8eff-e370097f0726"
+  tunnel: "f8c06a8a-1880-4e6e-8502-deb8f1d1253b"
```

Comment:
This is was wrong. `tunnelIDs` does not exist and `tunnel` is a single string. So now you need two releases to create two tunnels. We can improve in the future but it is how works now